packages/block-editor/src/components/link-control/index.js

```diff
@@ -181,6 +183,7 @@ function LinkControl( {
 			onClose={ closeLinkUI }
 			position="bottom center"
 			focusOnMount="firstElement"
+			{ ...popoverProps }
```

Comment:
This seems logical.

Reply:
I actually think, there's something wrong in the control for now. I feel instead of directly being a "Popover", we need two components:

 - LinkControl,  which is an inline UI to edit links. (like all controls, TextControl, SelectControl, CustomSelectControl... )
 - LinkControlPopover, potentially this is useless and can be easily replaced with just Popover wrapping LinkControl.

Reply:
Yes, `LinkControl` should be independent of `Popover`. You should be able to render the control wherever you want.

This PR or another one @youknowriad ?

Reply:
Let's keep it for another one.

Reply:
It's a bit weird that we grant that we want to remove the Popover rendering, in the same pull request where we're adding a new `popoverProps` prop. I guess we don't think it's an issue in that the component is already "experimental" anyways?

Reply:
Exactly, that was my thinking. Trying to limit the changes to implement this and separate the Popover from the Control in its own PR.